### PR TITLE
Discoball (failed)

### PR DESCRIPTION
**Pull request by rlugge**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -11,4 +11,6 @@ gemspec
 # your gem to rubygems.org.
 
 # To use a debugger
-# gem 'byebug', group: [:development, :test]
+gem 'byebug', group: [:development, :test]
```
Comment (dphaener) [inline Gemfile]: I prefer to use the gemspec file for adding gem dependencies. If they are in this file, the gem will still work as expected, but rubygems (and other gem hosts) won't pick it up as a dependency.

Same goes for below.

**Comment by dphaener** [comment]
@rlugge Can you push what you have right now so I can take a look? Want to make sure we're on the same page.

File: lib/montage_rails/mock.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    before do
+      content_type :json
+    end
+
+    get "/api/v1/schemas/:schema" do
+      klass = "#{params[:schema].classify}Resource".constantize
+      klass.schema_definition.to_json
+    end
+  end
+end
```
Comment (dphaener) [inline lib/montage_rails/mock.rb]: Let's go ahead and remove this file if it's not needed anymore.

File: test/dummy/config/routes.rb [hunk 3]
```diff
@@ -1,56 +1,5 @@
...
-
-  # Example of regular route:
-  #   get 'products/:id' => 'catalog#view'
-
-  # Example of named route that can be invoked with purchase_url(id: product.id)
-  #   get 'products/:id/purchase' => 'catalog#purchase', as: :purchase
-
-  # Example resource route (maps HTTP verbs to controller actions automatically):
-  #   resources :products
+require 'montage_rails/mock'
```
Comment (dphaener) [inline test/dummy/config/routes.rb]: I think this should point to `montage_rails/mock_server` now correct?

Comment (rlugge) [inline test/dummy/config/routes.rb]: It should already be pointing to montage_rails/mock_server, actually.  Checking my code to see if I failed to commit something.

**Pull request closed without merging**